app/uk/gov/hmrc/testuser/models/Payloads.scala

```diff
@@ -16,16 +16,21 @@
 
 package uk.gov.hmrc.testuser.models
 
+case class AuthenticationRequest(username: String, password: String)
 
 object ErrorCode extends Enumeration {
 
   type ErrorCode = Value
 
   val INTERNAL_SERVER_ERROR = Value("INTERNAL_SERVER_ERROR")
+  val WRONG_PASSWORD = Value("WRONG_PASSWORD")
+  val USERNAME_NOT_FOUND = Value("USERNAME_NOT_FOUND")
```

Comment:
we usually only return an INVALID_CREDENTIALS when wrong username or password to avoid giving too much information about the failure.